vcbuild.bat

```diff
@@ -301,7 +304,12 @@ set "msbcpu=/m:2"
 if "%NUMBER_OF_PROCESSORS%"=="1" set "msbcpu=/m:1"
 set "msbplatform=Win32"
 if "%target_arch%"=="x64" set "msbplatform=x64"
-if "%target%"=="Build" if defined no_cctest set target=node
+if "%target%"=="Build" (
+  if defined no_cctest set target=rename_node_bin_win
+  if "%test_args%"=="" set target=rename_node_bin_win 
+  if defined cctest set target="Build"
+)
+if "%target%"=="rename_node_bin_win" if exist "%config%\cctest.exe" del "%config%\cctest.exe"
```

Comment:
@lundibundi not sure I understand your question, `target` might have just been changed above. This uses `rename_node_bin_win` instead of just `node` to be compatible with `vcbuild dll`, but should be the same otherwise.

Reply:
Yeah `rename_node_bin_win` is an old kludge.
https://github.com/nodejs/node/blob/d1a55d329a0039061915f904baafb1abfced22ab/node.gyp#L834-L842
I have a refactor for this in the pipeline.